Automator/AutoWrapper.py

```diff
@@ -14,8 +14,8 @@ def main():
 
     start = time.time()
 
-    maze_dir = "../Mazes/"
-    image_dir = "../Images/"
+    maze_dir = "../Worlds/"
```

Comment:
This should be `../Mazes` you might need to pull my recent changes and then move your local files into that directory.